### PR TITLE
Feature/release prep

### PR DESCRIPTION
**Pull request by jpaidoussi**

### CODE REVIEW

File: seed_message_sender/settings.py [hunk 1]
```diff
@@ -124,7 +136,6 @@
 
 # REST Framework conf defaults
 REST_FRAMEWORK = {
-    'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAdminUser',),
```
Comment (JayH5) [inline seed_message_sender/settings.py:127]: This seems important.. I don't know what it's doing exactly..

File: README.md [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-
-## Metrics
-##### inbounds.created.sum
-`sum` Total number of inbounds created
-
-##### vumimessage.tries.sum
-`sum` Total number of outbound message send tries
-
-##### vumimessage.maxretries.sum
-`sum` Total number of outbound message send attempts that reached max retries
```
Comment (JayH5) [inline README.md:17]: Is this information anywhere else? Is it up-to-date?

**Review by JayH5 — APPROVED** [review]
:+1: on the packaging side of things.. the app level stuff maybe somebody else should look at

File: README.md (hunk 2 above)
Comment (jpaidoussi) [reply README.md:17]: It isn't up to date and there is a ticket to create formal documentation for all the services that haven't already gotten some.

File: seed_message_sender/settings.py (hunk 1 above)
Comment (jpaidoussi) [reply seed_message_sender/settings.py:127]: This is removing a duplicated directive that is defined a few lines below and uses the correct class: rest_framework.permissions.IsAuthenticated

**Pull request merged**